src/rules/no-focused-tests.ts

```diff
@@ -0,0 +1,85 @@
+import {
+  AST_NODE_TYPES,
+  TSESTree,
+} from '@typescript-eslint/experimental-utils';
+import { DescribeAlias, TestCaseName, createRule } from './tsUtils';
```

Comment:
You forgot to export these new ones

![image](https://user-images.githubusercontent.com/1404810/61576087-7a697b80-aad5-11e9-81a9-66175e8d722d.png)

Reply:
O.o I feel like I need to [commit sudoku](https://knowyourmeme.com/memes/commit-sudoku) - how did I miss that...?

Reply:
Actually no it just needs to be merged w/ `reapply-ts` :)

Reply:
ah, you've cherry-picked. Makes sense 🙂

Reply:
Yeah it got lost in the shuffle. The whole guard thing + timezones threw me for a loop :joy: